Install.ps1

```diff
@@ -0,0 +1,63 @@
+$ErrorActionPreference = "Stop"
+
+if ([Environment]::Is64BitOperatingSystem -eq $false)
+{
+    Write-Output "Chia requires a 64-bit Windows installation"
+    Exit 1
+}
+
+if ((Get-Item "$env:windir\System32\msvcp140.dll").Exists -eq $false)
+{
+    Write-Output "Unable to find Visual C++ Runtime DLLs"
+    Write-Output ""
+    Write-Output "Download and install the Visual C++ Redistributable for Visual Studio 2019 package from:"
+    Write-Output "https://visualstudio.microsoft.com/downloads/#microsoft-visual-c-redistributable-for-visual-studio-2019"
+    Exit 1
+}
+
+if ($null -eq (Get-Command git -ErrorAction SilentlyContinue))
+{
+    Write-Output "Unable to find git"
+    Exit 1
+}
+
+git submodule update --init mozilla-ca
+
+if ($null -eq (Get-Command py -ErrorAction SilentlyContinue))
+{
+    Write-Output "Unable to find py"
+    Write-Output "Note the check box during installation of Python to install the Python Launcher for Windows."
+    Write-Output ""
+    Write-Output "https://docs.python.org/3/using/windows.html#installation-steps"
+    Exit 1
+}
+
+$pythonVersion = (py --version).split(" ")[1]
+if ([version]$pythonVersion -lt [version]"3.7.0")
+{
+    Write-Output "Found Python version:" $pythonVersion
```

Comment:
I was thinking immediately above the `if` so it was shown in both cases.  Not a big deal though.